vignettes/available-theming-variables.Rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: "Available theming variables"
+output: rmarkdown::html_vignette
+author: "Nick Strayer"
+date: "`r Sys.Date()`"
+vignette: >
+  %\VignetteIndexEntry{available-theming-variables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  echo = FALSE, 
+  message = FALSE
+)
+
+
+htmltools::tagList(rmarkdown::html_dependency_font_awesome(), rmarkdown::html_dependency_bootstrap("default"))
```

Comment:
Oh wow, TIL pkgdown provides Bootstrap JS but not the CSS (only if you've specified a Bootswatch theme)